crawler/dockercontainer.py

```diff
@@ -23,6 +24,11 @@
                                exec_dockerinspect,
                                poll_container_create_events)
 
+try:
+    basestring        # Python 2
```

Comment:
* basestring is a builtin in Python 2 but it was removed in Python 3.
* basestring unified str and unicode in Python 2.
* ALL strs are unicode in Python 3 so both unicode and basestring were dropped from Python 3.